### DISCUSSION TITLE
求大佬解答

### DISCUSSION THREAD

**Discussion by potter645** [ZelKnow/Hearthbot#11] [Q&A] [answered]
运行bot报错
'NoneType' object is not iterable
line 22, in <module>
    nonebot.load_all_plugins(set(config.plugins), set(config.plugin_dirs))

**Comment by ZelKnow** [comment, marked as answer]
感觉应该是没有 `.env` 文件或是 `.env` 没有配置好。参考[这里](https://github.com/ZelKnow/Hearthbot/blob/master/docs/installation.md#%E9%85%8D%E7%BD%AE)进行配置，或是直接使用 `.env.example`

**Reply by potter645** [reply]
果然是这样，感谢大佬
